tests/PerformanceTests/PerformanceTestService/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Identity.Web;
+using Microsoft.Identity.Web.Test.Common;
+
+namespace PerformanceTestService.Controllers
+{
+    [ApiController]
+    [Authorize]
+    [Route("SecurePage")]
+    public class WeatherForecastController : ControllerBase
```

Comment:
We probably want to rename the file and the class tio SecurePageController ?

Reply:
Yea, I'll refactor the naming.